app/views/transactions/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Transactions</h1>
+
```

Comment:
Please add an option at the top of the transaction page where the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/ror/capstone/images/transactions_list.png